composite/apy-finance/src/abi/ERC20.json

```diff
@@ -0,0 +1,222 @@
+[
```

Comment:
We should be able to include this from a public package as this is a standard interface. Perhaps from `@openzeppelin/contracts/build/contracts/ERC20.json`

Reply:
Is it possible that this `abi` doesn't have `decimals` included?

Reply:
[It should](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/5a58fd2de01ac4292510a9f0b30d52a9b4ef8ecd/contracts/token/ERC20/ERC20.sol#L44), why do you ask?